editor/src/components/canvas/canvas-strategies/strategies/flow-reorder-strategy.spec.browser2.tsx

```diff
@@ -428,4 +499,31 @@ describe('Flow Reorder Strategy (Mixed Display Type)', () => {
 `),
     )
   })
+
+  describe('with fragments as siblings', () => {
+    it('simple dragging the element in a block reorders it', async () => {
+      const renderResult = await renderTestEditorWithCode(
+        makeTestProjectCodeWithSnippet(TestProjectBlockElementsWithFragment()),
+        'await-first-dom-report',
+      )
+
+      // drag element 'CCC' up a little to replace it with it's direct sibling
+      const dragDelta = windowPoint({ x: 0, y: -45 })
+      await dragElement(renderResult, 'ccc', dragDelta, emptyModifiers, [
+        'utopia-storyboard-uid/scene-aaa',
+        'utopia-storyboard-uid/scene-aaa/app-entity',
+        'utopia-storyboard-uid/scene-aaa/app-entity:container',
+        'utopia-storyboard-uid/scene-aaa/app-entity:container/aaa',
+        'utopia-storyboard-uid/scene-aaa/app-entity:container/38e/bbb',
+        'utopia-storyboard-uid/scene-aaa/app-entity:container/ccc',
+      ])
+
+      await renderResult.getDispatchFollowUpActionsFinished()
+
+      await wait(10000)
```

Comment:
10 seconds!

Reply:
I forgot to delete this after debugging tests, I removed them in https://github.com/concrete-utopia/utopia/pull/3302/commits/a01cb2d89c9dd09f7b47016a21b4c70d2256b64b